test_matrix.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+from Matrix import Matrix
+
+
+class Test_matrix(unittest.TestCase):
+    def setUp(self):
+        self.x = Matrix(((1, 2), (3, 4)))
+        self.y = Matrix(((4, 3), (2, 1)))
+        self.zeros = Matrix(((0, 0), (0, 0)))
+        self.tens = Matrix(((10, 10), (10, 10)))
+        self.negative = Matrix(((-1, -1), (-1, -1)))
+        self.three = Matrix(((1, 2, 0), (0, 1, 2), (2, 0, 1)))
+        self.half = Matrix(((0.5, 0.5), (0.5, 0.5)))
```

Comment:
All those matrix you are creating here would be recreated before any test is running, try and create here only those you would need for **all** the tests.

Reply:
fix